docker/docker-compose.yml

```diff
@@ -39,7 +39,7 @@ services:
     ports:
       - "9130:8080"
     volumes:
-      - ../src/test/resources/mappings:/home/wiremock/mappings
+      - ./mappings:/home/wiremock/mappings
```

Comment:
I think it's references to test folder to not change mappings in two places

Reply:
In this case there is some userSchema error raises when calling wiremock
I mean when I run docker-compose and then try to create authority source file on my local machine for example